### ISSUE TITLE
review next doc sweep/update

### ISSUE THREAD

**Issue by CeciAc** [aspnet/AspNetCore.Docs#11299]
@DHclly commented on [Wed Mar 06 2019](https://github.com/aspnet/Docs.zh-cn/issues/119)

before:
```
        public DbSet<Student> Student { get; set; }
        public DbSet<Enrollment> Enrollment { get; set; }
        public DbSet<Course> Course { get; set; }
```
after:
```
        public DbSet<Course> Courses { get; set; }
        public DbSet<Enrollment> Enrollment { get; set; }
        public DbSet<Student> Student { get; set; }
        public DbSet<Department> Departments { get; set; }
        public DbSet<Instructor> Instructors { get; set; }
        public DbSet<OfficeAssignment> OfficeAssignments { get; set; }
        public DbSet<CourseAssignment> CourseAssignments { get; set; }
```
why not have same format ?
link:https://github.com/aspnet/Docs/blob/6ddd8a7675c1c1d997c8ab2d4498538e44954cac/aspnetcore/data/ef-rp/intro/samples/cu21/Data/SchoolContext.cs#L71

---
#### 文档详细信息

⚠ *请勿编辑此部分。 docs.microsoft.com ➟ GitHub 问题链接必须具有此部分。*

* ID: 634dfb2c-2626-9ec5-35fe-2bd2f27c5e23
* Version Independent ID: 7a77b058-f005-8d68-ae46-e487c597f18e
* Content: [ASP.NET Core 中的 Razor 页面和 EF Core - 数据模型 - 第 5 个教程（共 8 个）](https://docs.microsoft.com/zh-cn/aspnet/core/data/ef-rp/complex-data-model?view=aspnetcore-2.2&tabs=visual-studio#feedback)
* Content Source: [aspnetcore/data/ef-rp/complex-data-model.md](https://github.com/aspnet/docs.zh-cn/blob/live/aspnetcore/data/ef-rp/complex-data-model.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by CeciAc** [comment]
Hi team, 
the user is reporting a typo in the example 
best

**Comment by Rick-Anderson** [comment]
@tdykstra  can I close this? I don't see the problem?

**Comment by tdykstra** [comment]
It’s a valid point, I’ll fix it when I do the 3.0 update

**Comment by DHclly** [comment]
> @tdykstra can I close this? I don't see the problem?

eh , I'm sorry that I dont't have a detail description , this is a small problem , I mean is that some property is plural , end with 's' , and some is singular , maybe have the same format will better.

for example , the property `course` ,  at before is `public DbSet<Course> Course { get; set; }` ，at after is `public DbSet<Course> Courses { get; set; }` ， I think it should keep same in tutorial 。

```c#
public DbSet<Course> Courses { get; set; } // the before demo code should be like this
public DbSet<Enrollment> Enrollment { get; set; } //Enrollment  =>Enrollments
public DbSet<Student> Student { get; set; } // Student =>Students
public DbSet<Department> Departments { get; set; }
public DbSet<Instructor> Instructors { get; set; }
public DbSet<OfficeAssignment> OfficeAssignments { get; set; }
public DbSet<CourseAssignment> CourseAssignments { get; set; }
```
